<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ClosePeriod_Error</class>
 <widget class="QDialog" name="Dialog_ClosePeriod_Error">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Close Period 0</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>260</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Yes</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_txt_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>There are missing decisions.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_txt_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>25</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Do you really want to do it?</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>111</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>222</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>333</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>444</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>555</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>666</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>777</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>888</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_ClosePeriod_Error</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_ClosePeriod_Error</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
